common/ai_strategy/GRE.txt

```diff
@@ -40,555 +40,6 @@ GRE_crete_is_a_bastion = {
 	}
 }
 
-GRE_befriend_balkan_countries_YUG = {
```

Comment:
While you're at it, can you tell Greece to stay neutral until they get invaded by italy? I keep seeing them join the Allies early because they are democratic and get in the war before the italian invasion.